docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -444,7 +444,7 @@ This can be useful for accessing native features which are not available in
 Trino or for improving query performance in situations where running a query
 natively may be faster.
 
-.. include:: polymorphic-table-function-ordering.fragment
+.. include:: query-table-function-ordering.fragment
```

Comment:
This fragment does not apply to Elasticsearch. There's no `order by` in the query syntax.

Reply:
Okay but should we just reword then so it drops the "order by" syntax .. I assume results can still come from ES as ordered data...

Reply:
Indeed the ES result can be ordered, so the fragment is somehow relevant. However, I think that we could skip it here altogether. With other functions, the passed query could contain "ORDER BY", and that could mislead users into thinking that the overall query result should be ordered. WIth ES this is not the case. I would draw the line here. Otherwise, we might end up adding the "ordering disclaimer" to all table functions. Instead, we should try to document the underlying general rule, that is: the results from any table-like construct (table, subquery, PTF) are inherently unordered.